conda/recipes/libcudf/meta.yaml

```diff
@@ -68,6 +68,7 @@ requirements:
     - dlpack {{ dlpack_version }}
     - librdkafka {{ librdkafka_version }}
     - fmt {{ fmt_version }}
+    - flatbuffers
```

Comment:
Add an entry in `conda_build_config.yaml` with version 24.3.25 and use it here like `flatbuffers {{ flatbuffers_version }}`.